<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>prod_displayClass</class>
 <widget class="QDialog" name="prod_displayClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2065</width>
    <height>899</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>prod_display</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <widget class="QTableWidget" name="elect_tableWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>481</width>
     <height>621</height>
    </rect>
   </property>
   <property name="columnCount">
    <number>4</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>120</number>
   </attribute>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QTableWidget" name="groc_tableWidget">
   <property name="geometry">
    <rect>
     <x>1220</x>
     <y>150</y>
     <width>481</width>
     <height>621</height>
    </rect>
   </property>
   <property name="columnCount">
    <number>4</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>120</number>
   </attribute>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QTableWidget" name="cloth_tableWidget">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>150</y>
     <width>481</width>
     <height>621</height>
    </rect>
   </property>
   <property name="columnCount">
    <number>4</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>120</number>
   </attribute>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ELECTRONICS ITEMS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1230</x>
     <y>110</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>GROCERY ITEMS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>110</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CLOTHING ITEMS</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
